src/index.ts

```diff
@@ -19,6 +19,10 @@ import React from 'react';
  * - `['a', 1, 1000]`
  * - `['a', 2]`
  */
+// Do not try to use useLayoutEffect if DOM not available (SSR)
+const isDOMLoaded = typeof window !== 'undefined';
+const useIsomorphicLayoutEffect = isDOMLoaded ? React.useLayoutEffect : React.useEffect;
+
```

Comment:
1. Please move it down and place just before useSubscribedStateMethods function.
2. isDOMLoaded is unnecessary to be kept as a variable. Just use the condition directly: `const useIsomorphicLayoutEffect = typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;`

Reply:
sure will update it.